### ISSUE TITLE
navi --cheatsh tar returns navi was unable to call wget on macOS.

### ISSUE THREAD

**Issue by shinokada** [denisidoro/navi#457]
**Describe the bug**

## macOS version
![image](https://user-images.githubusercontent.com/147320/107009127-f6bad100-67d7-11eb-9717-7fe3dcd6a664.png)

## Shell versions

```
❯ sh --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin20)
Copyright (C) 2007 Free Software Foundation, Inc.

❯ bash --version
GNU bash, version 5.1.4(1)-release (x86_64-apple-darwin20.2.0)
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

## navi version

```
navi --version
navi 2.14.0
```

I installed the navi using `brew install navi`.

## Problems

1. When I run `navi --cheatsh tar` I get the following error.

```
navi --cheatsh tar
navi was unable to call wget.
Make sure wget is correctly installed.
Failed to read /dev/tty
```

In the document, it doesn't say that I have to install wget.

2. Using tldr also outputs an error.

```
❯ navi --tldr docker
navi was unable to call tldr.
Make sure tldr is correctly installed.
Refer to https://github.com/tldr-pages/tldr for more info.

Note:
The tldr client written in C (the default one in Homebrew) doesn't support markdown files, so navi can't use it.
The client written in Rust is recommended. The one available in npm works, too.

Failed to read /dev/tty
```

How can I use cheatsh and tldr on macOS?

**Comment by denisidoro** [comment]
`brew install wget` and `npm install -g tldr` should do the trick

**Comment by denisidoro** [comment]
These dependencies aren't shipped with navi because not everyone uses the `--tldr` and `--cheatsh` flags.
